code/modules/events/wizard/rpgloot.dm

```diff
@@ -21,7 +23,38 @@
 		else
 			I.name = "[I.name] of [pick(suffixes)]"
 
-		I.force 		+= quality
-		I.force			= max(0,I.force)
-		I.throwforce 	+= quality
-		I.throwforce	= max(0,I.throwforce)
\ No newline at end of file
+		I.force 		= max(0,I.force + quality)
+		I.throwforce	= max(0,I.throwforce + quality)
+		I.armor			+= quality
```

Comment:
I just tried this, it does not work. It just tacks on a number to the end of the list.